### PR TITLE
Added the PIM policies article

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by msewaweru** [comment]
@mwongerapk, please review this.

File: microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+ms.topic: how-to
+ms.date: 09/12/2022
+author: msewaweru
+manager: mwongerapk
+ms.author: eunicewaweru
+---
+
+# Manage PIM policies in Microsoft Graph PowerShell
+
+The Azure AD Privileged Identity Management (PIM) service allows role administrators to make time-bound admin role assignments. Each role has a set of settings that manage it. For example, activation and assignment settings. These role settings are called policies. Each policy has a set of rules.
```
Comment (mwongerapk) [inline microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md:14]: Perhaps, group of settings?

File: microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    Select **Consent on behalf of your organization** before accepting in the sign-in dialog box.
+
+## Step 1: List policies
+
+### List all policies
+
+To list all policies at directory level, run:
+
+```powershell
+Get-MgPolicyRoleManagementPolicy -Filter "scopeId eq '/' and scopeType eq 'DirectoryRole'"
```
Comment (mwongerapk) [inline microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md:41]: In Step 2, I notice we define what the cmdlet returns then give the snippet. Should we do the same for step 1?

File: microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03_9b895d92-2cd3-44c7-9d02-a6ac2d5ea5c3 Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03 9b895d92-2cd3-44c7-9d02-a6ac2d5ea5c3 /       Directory
+```
+
+## Step 3: Get policy rules
+
+### Get all rules for a policy
+
+To get all the rules for a policy, run:
+
+```powershell
```
Comment (mwongerapk) [inline microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md:113]: Similarly for step 3, adding a description the cmdlet

File: microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+```Output
+
+Id                   : Expiration_EndUser_Assignment
+Target               : Microsoft.Graph.PowerShell.Models.MicrosoftGraphUnifiedRoleManagementPolicyRuleTarget
+AdditionalProperties : {[@odata.context, https://graph.microsoft.com/v1.0/$metadata#policies/roleManagementPolicies('Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03')/rules/$entity], [@odata.type, #microsoft.graph.unifiedRoleManagementPolicyExpirationRule],
+                       [isExpirationRequired, True], [maximumDuration, PT8H]}
+```
+
+## Step 4: Update a policy rule 
+
```
Comment (mwongerapk) [inline microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md:150]: Is this just an example, and the same steps should be followed for other rules they need to update? We could perhaps express that more clearly if that is the case

File: microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md (hunk 2 above)
Comment (msewaweru) [reply microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md:41]: I have removed the cmdlet definition and instead defined what assignments are. Thanks.

**Review by mandardalvi — APPROVED** [review]

**Comment by msewaweru** [comment]
@timayabi2020, please merge this.

**Pull request merged**